### PR TITLE
Update Proposed-WG-WebIdentityCredentials.md

### PR DESCRIPTION
**Pull request by judielaine**
Added an an "and"  and specified  a particular SAML profile. Added the "authentication intermediaries" to address the hops needed in enterprise, healthcare, banking, and higher ed.

### CODE REVIEW

File: charters/Proposed-WG-WebIdentityCredentials.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 The *W3C Web Identity Credential Working Group* will develop
 recommendation-track specifications defining an API that allows websites
-to request a federated identity credential or assertion with the purpose of authenticating a user and/or requesting a set of claims in a compatible way OIDC or SAML.
+to request a federated identity credential or assertion with the purpose of authenticating a user and/or requesting a set of claims compatible with OIDC and SAML (specifically the [SP initiated Web Browser SSO profile](http://docs.oasis-open.org/security/saml/v2.0/saml-profiles-2.0-os.pdf)).
```
Comment (TallTed) [inline charters/Proposed-WG-WebIdentityCredentials.md:5]:
```suggestion
to request a federated identity credential or assertion with the purpose
of authenticating a user and/or requesting a set of claims compatible with
OIDC and SAML, specifically the
[SP initiated Web Browser SSO profile](http://docs.oasis-open.org/security/saml/v2.0/saml-profiles-2.0-os.pdf).
```

File: charters/Proposed-WG-WebIdentityCredentials.md [hunk 2]
```diff
@@ -19,11 +19,11 @@ to request a federated identity credential or assertion with the purpose of auth
 The Working Group will specify new web platform features intended to be
 implemented in browsers or similar user agents. The purpose of these
 features is to support authentication and authorization flows without
-compromising security principles for IdPs, RPs, and user agents as well
+compromising security principles for IdPs, RPs,  authentication intermediaries (such as most of the industry-adopted solutions for MFA and Passwordless technologies), and user agents as well
```
Comment (TallTed) [inline charters/Proposed-WG-WebIdentityCredentials.md:22]:
```suggestion
compromising security principles for IdPs, RPs,  authentication intermediaries
(such as most of the industry-adopted solutions for MFA and Passwordless
technologies), and user agents as well
```

File: charters/Proposed-WG-WebIdentityCredentials.md [hunk 3]
```diff
@@ -19,11 +19,11 @@ to request a federated identity credential or assertion with the purpose of auth
...
 as user privacy. Here \"privacy\" minimally refers to the appropriate
 processing of personal information. The result of this work is the
 development of a new mechanism and how information is passed by the
-browser between the RP and the IdP to facilitate federated
+browser between the RP, the IdP,  and authentication intermediaries,  to facilitate federated
 authentication; it is not an authentication method.
 
 If the mechanisms developed to support authentication and authorization flows look like they will result in breaking changes for existing protocols, work on that mechanism must include a well-documented transition period.
```
Comment (TallTed) [inline charters/Proposed-WG-WebIdentityCredentials.md:29]:
```suggestion
If the mechanisms developed to support authentication and authorization
flows look like they will result in breaking changes for existing
protocols, work on that mechanism must include a well-documented
transition period.
```

File: charters/Proposed-WG-WebIdentityCredentials.md (hunk 1 above)
Comment (judielaine) [reply charters/Proposed-WG-WebIdentityCredentials.md:5]: There are multiple OIDC profiles: not sure which ones should be considered in scope. The relevant SAML profile is clear.

File: charters/Proposed-WG-WebIdentityCredentials.md [hunk 4]
```diff
@@ -19,11 +19,11 @@ to request a federated identity credential or assertion with the purpose of auth
 The Working Group will specify new web platform features intended to be
 implemented in browsers or similar user agents. The purpose of these
 features is to support authentication and authorization flows without
-compromising security principles for IdPs, RPs, and user agents as well
+compromising security principles for IdPs, RPs,  authentication intermediaries (such as most of the industry-adopted solutions for MFA and Passwordless technologies), and user agents as well
 as user privacy. Here \"privacy\" minimally refers to the appropriate
 processing of personal information. The result of this work is the
 development of a new mechanism and how information is passed by the
-browser between the RP and the IdP to facilitate federated
+browser between the RP, the IdP,  and authentication intermediaries,  to facilitate federated
```
Comment (pascoej) [inline charters/Proposed-WG-WebIdentityCredentials.md:26]:
```suggestion
browser between the RP, the IdP, and authentication intermediaries, to facilitate federated
```

**Review by martinthomson — CHANGES_REQUESTED** [review]
I like the addition of the intermediaries here, but the additional detail is unnecessary.  That means the profile link and the parenthetical.

File: charters/Proposed-WG-WebIdentityCredentials.md (hunk 1 above)
Comment (timcappalli) [reply charters/Proposed-WG-WebIdentityCredentials.md:5]: consensus at TPAC that the "with/and" change is fine, but adding protocol specifics (the parenthetic text) should be avoided.

Comment (TallTed) [reply charters/Proposed-WG-WebIdentityCredentials.md:5]: Acting on @timcappalli's note
```suggestion
to request a federated identity credential or assertion with the purpose
of authenticating a user and/or requesting a set of claims compatible with
OIDC and SAML.
```

Comment (judielaine) [reply charters/Proposed-WG-WebIdentityCredentials.md]: Forgive my GIT clumsiness  -- has committing Tall Ted's suggestion done the job?

**Comment by asr-enid** [comment]
Suggestion to add intermediaries has been merged here https://github.com/fedidcg/fedidcg.github.io/pull/18 based on CG consensus, other changes have been determined to be too specific for this charter and are generally covered with the existing wording

**Pull request closed without merging**